src/Raven.Server/Documents/PeriodicExport/PeriodicExportRunner.cs

```diff
@@ -402,19 +406,8 @@ private void WriteStatus()
             using (_database.DocumentsStorage.ContextPool.AllocateOperationContext(out context))
             using (var tx = context.OpenWriteTransaction())
             {
-                var status = new DynamicJsonValue
-                {
-                    ["LastDocsEtag"] = _status.LastDocsEtag,
-                    ["LastExportAt"] = _status.LastExportAt.GetDefaultRavenFormat(),
-                    ["LastFullExportAt"] = _status.LastFullExportAt.GetDefaultRavenFormat(),
-                    ["LastFullExportDirectory"] = _status.LastFullExportDirectory
-                };
-                var readerObject = context.ReadObject(status, Constants.Documents.PeriodicExport.StatusKey, BlittableJsonDocumentBuilder.UsageMode.ToDisk);
-                var putResult = _database.DocumentsStorage.Put(context, Constants.Documents.PeriodicExport.StatusKey, null, readerObject);
+                _database.DocumentsStorage.SetDatabasePeriodicExportStatus(context, _status);
```

Comment:
i can take a snapshot, this was the design of bundles, i did minor changes to the design to integrate the code, i haven't refactored the logic yet.
We will have to have step two of bundles and raft after heartbeat is implemented i'll have to adjust the code to select active bundle runner and such.